lms/djangoapps/verify_student/tests/test_views.py

```diff
@@ -2105,6 +2108,13 @@ def setUp(self):
         )
 
         self.attempt = SoftwareSecurePhotoVerification.objects.filter(user=self.user)
+        location_id = VerificationStatus.get_location_id(self.attempt)
```

Comment:
Yes there are few conflicts. That's why i will wait for zubair's PR merge and then i will rebase and merge.